src/pages/waiting-room-to-car/cat-be/waiting-room-to-car.cat-be.page.ts

```diff
@@ -0,0 +1,328 @@
+  getDualControls,
+  isAutomatic,
+  isManual,
+} from '../../../modules/tests/vehicle-details/vehicle-details.selector';
+import {
+  getInstructorAccompaniment,
+  getSupervisorAccompaniment,
+  getOtherAccompaniment,
+  getInterpreterAccompaniment,
+} from '../../../modules/tests/accompaniment/accompaniment.selector';
+import { getCandidate } from '../../../modules/tests/candidate/candidate.reducer';
+import { getUntitledCandidateName } from '../../../modules/tests/candidate/candidate.selector';
+import { getTests } from '../../../modules/tests/tests.reducer';
+import { FormGroup } from '@angular/forms';
+import { TellMeQuestion } from '../../../providers/question/tell-me-question.model';
+import { QuestionProvider } from '../../../providers/question/question';
+import { getInstructorDetails } from '../../../modules/tests/instructor-details/instructor-details.reducer';
+import { getInstructorRegistrationNumber } from '../../../modules/tests/instructor-details/instructor-details.selector';
+import {
+  TellMeQuestionSelected,
+  TellMeQuestionCorrect,
+  TellMeQuestionDrivingFault,
+  QuestionOutcomes,
+  EyesightTestReset,
+  EyesightTestPassed,
+  EyesightTestFailed,
+} from '../../../modules/tests/test-data/test-data.actions';
+import {
+  isTellMeQuestionSelected,
+  isTellMeQuestionDrivingFault,
+  isTellMeQuestionCorrect,
+  tellMeQuestionOutcome,
+  getVehicleChecks,
+  getTellMeQuestion,
+  hasEyesightTestBeenCompleted,
+  hasEyesightTestGotSeriousFault,
+} from '../../../modules/tests/test-data/test-data.selector';
+import { getTestData } from '../../../modules/tests/test-data/test-data.reducer';
+import { PersistTests } from '../../../modules/tests/tests.actions';
+import { CAT_B, CAT_BE } from '../../page-names.constants';
+import { BasePageComponent } from '../../../shared/classes/base-page';
+
+interface WaitingRoomToCarPageState {
+  candidateName$: Observable<string>;
+  registrationNumber$: Observable<string>;
+  instructorRegistrationNumber$: Observable<number>;
+  transmission$: Observable<GearboxCategory>;
+  schoolCar$: Observable<boolean>;
+  dualControls$: Observable<boolean>;
+  instructorAccompaniment$: Observable<boolean>;
+  supervisorAccompaniment$: Observable<boolean>;
+  otherAccompaniment$: Observable<boolean>;
+  interpreterAccompaniment$: Observable<boolean>;
+  eyesightTestComplete$: Observable<boolean>;
+  eyesightTestFailed$: Observable<boolean>;
+  gearboxAutomaticRadioChecked$: Observable<boolean>;
+  gearboxManualRadioChecked$: Observable<boolean>;
+  tellMeQuestionSelected$: Observable<boolean>;
+  tellMeQuestionCorrect$: Observable<boolean>;
+  tellMeQuestionDrivingFault$: Observable<boolean>;
+  tellMeQuestionOutcome$: Observable<string>;
+  tellMeQuestion$: Observable<TellMeQuestion>;
+}
+
+@IonicPage()
+@Component({
+  selector: 'waiting-room-to-car-cat-be-page',
+  templateUrl: 'waiting-room-to-car.cat-be.page.html',
+})
+export class WaitingRoomToCarCatBEPage extends BasePageComponent {
```

Comment:
I think we are doing Be not BE - Can you double check

Reply:
Yup, we are :D, will change